### PR TITLE
Changed the build type to ament_python and Fixed package to run with ros2 run

### PR DESCRIPTION
**Pull request by ahcorde**
Fixed package to run with `ros2 run rqt_service_caller rqt_service_caller`. Related to this issue https://github.com/ros-visualization/rqt_console/issues/20

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    maintainer_email='dthomas@osrfoundation.org',
+    keywords=['ROS'],
+    classifiers=[
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Programming Language :: Python',
+        'Topic :: Software Development',
+    ],
+    description=(
+        'RQT plugin for monitoring ROS processes.'
```
Comment (dirk-thomas) [inline setup.py:31]: Copy-n-paste mistake.

**Comment by ahcorde** [comment]
Running CI up-to `rqt_service_caller`

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10907/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6259/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8882/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10800/)

**Comment by sloretz** [comment]
> One thing I noticed was that the "refresh" button does not have an icon; it's rendered as an empty button (I'm on Windows). I'm not sure if this is a bug introduced by this PR or an already existing bug. I'll try on Linux when I have a moment.

This bug is already present in 1.0.3 https://github.com/ros-visualization/rqt_service_caller/issues/19

**Comment by hidmic** [comment]
I verified it can be `ros2 run` on Windows. All icons show up.

Running CI up to `rqt_msg`, `rqt_srv`, `rqt_service_caller`, `rqt_top`, `rqt_shell`, `rqt_py_console`, `rqt_publisher`, `rqt_plot`:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/14396/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9170/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12071/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/14495/)

**Review by ivanpauno — APPROVED** [review]
LGTM!

Same comments than here https://github.com/ros-visualization/rqt_msg/pull/8#pullrequestreview-638063236.

**Pull request merged**